### DISCUSSION TITLE
How to use multiPassHashed

### DISCUSSION THREAD

**Discussion by MontejoJorge** [Ride-The-Lightning/RTL#1436] [General]
Hi, currently im using the APP_PASSWORD env to set the RTL login password. How i can set a hashed password?

**Comment by ShahanaFarooqui** [comment]
`APP_PASSWORD` is only allowed in plain text.

**Reply by MontejoJorge** [reply]
> `APP_PASSWORD` is only allowed in plain text.

Ye, thats why i want to use a hashed password. I see that its included in the `RTL-Config.json` but i dont know how to start using it. I have to manuali hash my password?
